packages/jest-runtime/src/index.ts

```diff
@@ -2161,6 +2180,9 @@ export default class Runtime {
       this.isolateModules(fn);
       return jestObject;
     };
+    const isolateModulesAsync = async(fn: () => void) => {
+      return this.isolateModulesAsync(fn);
+    }
```

Comment:
Seems to be redundant. The methods around return `jestObject`, but that is not the case case. I think it could be inlined.

Reply:
I think we could return a `Promise.resolve(jestObject)` there, and make it similar to the rest of the fluent interface of `jest`, WDYT?

Reply:
Can be, but is that useful? I mean, chaining something after `isolateModulesAsync()` will be clumsy. Perhaps it is fine to keep it as `Promise<void>` for now? If there is a clear use case, this can be changed at any time later.

Reply:
OK

Reply:
Done in 5810116f02db4f5b0ca78a0f49be40de0f925a1d